packages/react-tabster/package.json

```diff
@@ -34,7 +34,7 @@
     "@fluentui/react-theme": "9.0.0-rc.4",
     "@fluentui/react-utilities": "9.0.0-rc.5",
     "keyborg": "^1.1.0",
-    "tabster": "^1.1.1",
+    "tabster": "^1.2.2",
```

Comment:
Does this new version bring any breaking changes?

Reply:
Just making sure to avoid the same situation when the DOM structure was broken by an update, thanks for clarifying